api/src/test/java/org/openmrs/api/OrderServiceTest.java

```diff
@@ -56,43 +56,10 @@
 import org.junit.jupiter.api.Disabled;
 import org.junit.jupiter.api.Test;
 
+import org.openmrs.*;
```

Comment:
https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

Reply:
@dkayiwa  I have removed the wildcard imports